cli/src/awsbatch/utils.py

```diff
@@ -116,9 +115,6 @@ def get_job_type(job):
     """
     Get the type of the job.
 
-    Job type is of type string and not enum since enums have been introduced
```

Comment:
optional: Should we convert these string to enum? Since we already saitisfy the python version requirement.

Reply:
I am for keeping implementation as is, because converting from string to enum does not bring significant benefit to functionality